presto-tests/src/main/java/com/facebook/presto/tests/DistributedQueryRunner.java

```diff
@@ -116,7 +116,7 @@
 
     private final ReadWriteLock lock = new ReentrantReadWriteLock();
     private Optional<TestingPrestoServer> catalogServer = Optional.empty();
-    private Optional<List<TestingPrestoServer>> resourceManagers;
+    private Optional<List<TestingPrestoServer>> resourceManagers = Optional.empty();
```

Comment:
This change is not needed. `resourceManagers` is going to be assigned no matter what in the constructor.

Reply:
removed